Dockerfile

```diff
@@ -6,22 +6,27 @@ WORKDIR /go/src/github.com/openshift/machine-config-operator
 COPY . .
 # FIXME once we can depend on a new enough host that supports globs for COPY,
 # just use that.  For now we work around this by copying a tarball.
-RUN make install DESTDIR=./instroot && tar -C instroot -cf instroot.tar .
+ENV GOCACHE="/go/rhel8/.cache"
```

Comment:
I gave this and the RHEL9 stages separate caches so they don't step on each others toes and invalidate each other.